smos-report/package.yaml

```diff
@@ -0,0 +1,58 @@
+name: smos-report
+version: '0.0.0.0'
+category: Smos
+author: Nick Van den Broeck
+maintainer: syd@cs-syd.eu
+copyright: ! 'Copyright: (c) 2018 Tom Sydney Kerckhove'
+license: AllRightsReserved
+homepage: https://github.com/NorfairKing/smos
+
+executables:
+  smos-report:
+    main: Main.hs
+    source-dirs: app
+    ghc-options:
+    - -threaded
+    - -rtsopts
+    - -with-rtsopts=-N
+    dependencies:
+    - base
+    - smos-report
+
+library:
+  source-dirs: src/
+  ghc-options: -Wall
+  default-extensions: NoImplicitPrelude
+  dependencies:
+  - base
+  - configurator
```

Comment:
I'm not convinced that `configurator` is the right tool for the job.
`yaml` is probably simpler.

Reply:
Configurator is very easy to use, both in the Haskell code and for the configuration file, and config files is exactly what configurator is meant for.
Why do you believe yaml is simpler?